### ISSUE TITLE
Cant install themes

### ISSUE THREAD

**Issue by imanalien2020** [vercel/hyper#4662]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: 
- **Hyper.app version**: 
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: 
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: 

## Issue

Yes their is another thread with this issue from awhile ago it seems, as im new to command line and this app especially can someone please help me to get a theme installed. when i select a theme in the store and type the command to install it no matter what i get.."$..  or  hyper blah blah is not recognized as a command.." so since im new does anyone have a fix and if so please in laymans terms show me or tell me what to do to fix it and get a theme working please. Thank you.

**Comment by LabhanshAgrawal** [comment]
Plugins Menu -> Install Hyper CLI command in path
After this you should be able to do `hyper i theme`
